App.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Nodal - React Native App
+ * https://github.com/facebook/react-native
```

Comment:
while you are at it, remove line 3 and replace it with your project's GitHub link.